README.md

```diff
@@ -1,2 +1,177 @@
-# Hello-Microverse-Project
-The first project
+<a name="readme-top"></a>
+<div align="center">
+  <br/>
+
+  <h1><b>Hello Microvese Project</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
[OPTIONAL] Please, I suggest you remove unused markdown from your README file, as it makes the README file markdown noisy. 😀